src/dodal/devices/areadetector/plugins/MJPG.py

```diff
@@ -49,6 +50,9 @@ def _save_image(self, image: Image.Image):
 
         LOGGER.info(f"Saving image to {path}")
         image.save(path)
+
+        save_thumbnail(Path(path), image)
+
```

Comment:
PIL documentation states that this modifies the image, but looks like we are ok in all our usages.

Reply:
Yh, I thought about doing the copy but didn't seem worth it as we're always calling after saving the original. Thanks for checking though.